napari/utils/_tests/test_progress.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import pytest
+
+try:
+    import qtpy  # noqa
+except ImportError:
+    pytest.skip('Cannot test progress without qtpy.', allow_module_level=True)
+except RuntimeError:
+    pytest.skip(
+        'Cannot test progress without Qt bindings.', allow_module_level=True
+    )
```

Comment:
```suggestion
pytest.importorskip('qtpy', reason='Cannot test progress without qtpy.')
```

love that one :) 
https://docs.pytest.org/en/stable/reference.html#pytest-importorskip

I should have used it over in `test_magicgui` where I assume you got this piece of junk :joy:
(you can feel free to fix it over there too if you want to)